test.js

```diff
@@ -1,8 +1,376 @@
 import test from 'ava';
```

Comment:
Since all the tests are serial, you could use `import {serial as test} from 'ava';` and drop the serial property on the individual tests.